easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
```

Comment:
OK, the tests reports I got submitted here show that it does indeed work with CMakeMake.
@hajgato: Are you up for updating the Doxygen easyblock?